Cargo.toml

```diff
@@ -14,7 +14,7 @@ cargo = { git = "https://github.com/rust-lang/cargo" }
 env_logger = "0.4"
 failure = "0.1.1"
 jsonrpc-core = "8.0.1"
-languageserver-types = "0.16"
+languageserver-types = { git = "https://github.com/algesten/languageserver-types", branch="progress-params-016" }
```

Comment:
This is because the latest `languageserver-types` needs some work to integrate. I've done [this PR](https://github.com/gluon-lang/languageserver-types/pull/43) against languageserver-types master.

Reply:
Hmm, given that it might take a while for the next version of the LSP to be released, I would not wait for that and instead use a custom message with an issue and note to use this feature if/when it gets released

Reply:
The [changes](https://github.com/algesten/languageserver-types/commit/e6e86e5846698d503eb4f34d8e625f9e1af96271) for now can be put inside [lsp_data.rs](https://github.com/rust-lang-nursery/rls/blob/master/src/lsp_data.rs#L246)

Reply:
Alright. I make a local version.